pkg/cmd/db/search/search.go

```diff
@@ -122,134 +110,14 @@ func newDataVulnerabilityCmd() *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "vulnerability <Vulnerability ID>",
+		Use:   "vulnerability [<Vulnerability ID>]",
```

Comment:
```suggestion
		Use:   "vulnerability <Vulnerability ID>...",
```

When enclosed by [], it seems as optional as `[flags]`?
```
% go run ./cmd/vuls db search vulnerability -h
search data in vuls db by vulnerability id

Usage:
  vuls db search vulnerability [<Vulnerability ID>] [flags]
```

`man chmod` uses ... for one or more args:
```
NAME
       chmod - change file mode bits

SYNOPSIS
       chmod [OPTION]... MODE[,MODE]... FILE...
       chmod [OPTION]... OCTAL-MODE FILE...
       chmod [OPTION]... --reference=RFILE FILE...
```

How about that?